src/main/java/com/synopsys/integration/detect/workflow/ArtifactResolver.java

```diff
@@ -35,6 +35,7 @@
     private final ConnectionFactory connectionFactory;
     private final Gson gson;
 
+    //TODO- this is the class where we'll have logic regarding cached tools
```

Comment:
Is it? I don't see other changes to this class.
Or is the plan to move it here?